### PR TITLE
Show Trending Products and provided services on Home Page

### LINKED ISSUE #2: Create Home Page Of E-Commerce.
### Landing Page Should include - 

- # Header (it will be there for all pages.)
   - It will include the Logo/Name of E-Commerce and on clicking it this user should reach to Home Page irrespective of the current page.
   - A Search Bar where user can search for the products.
   - Login/Logout button
   - Navigation Links to Wishlist & Cart page.
 - # Categories
   - A list of all Categories of Products availaible.
- # Banner
  - A banner depicting the latest news/sale ofthe e-commerce.
- # Featured/Trending Products
  - The Top 3 Products from all products to showcase/give a brief about our products at home page itself.
- # Our Services
  - Some of the dummy services we provide such as Fast Delivery etc.

### PR DESCRIPTION
**Pull request by tushartiwari7**
# This PR includes last 2 tasks of #2 Issue.
- Here, I have added 3 Top Products of E-Commerce on Home Page under the Trending Products Section.
- Also, Some of the services provided by the E-commerce company to its consumers.
- Made it Responsive for mobile devices.
ss

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
In this PR, I like
that code is nicely written, and structured properly. proper use of folder structure.

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import './Header.css';
+import { Link } from 'react-router-dom';
+import {BsCart,BsHeart} from "react-icons/bs";
```
Comment (Priya-Patel31) [inline src/components/Header/Header.jsx:4]: Suggestion : You can have one file for all the react icons then import it from there .

File: src/components/FeaturedProducts/FeaturedProducts.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      }
+    })();
+  }, []);
+
+  return (
+    <section className="m-lg">
+      <h3 className="h2 ubuntu text-center mx-md my-lg">Trending Products</h3>
+      <section className="featured-products-wrapper flex wrap">
+        {featuredproducts.map((product) => (
+          <Card key={product._id} {...product} />
```
Comment (Priya-Patel31) [inline src/components/FeaturedProducts/FeaturedProducts.jsx:26]: Liked the way you have passed the props .

**Review by Priya-Patel31 — APPROVED** [review]
The code is clean and well structured . Really liked several things in your code . Just have few suggestions you can through it.

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (tushartiwari7) [reply src/components/Header/Header.jsx:4]: IMO I guess then that file will just have this line and won't be of much use.
```javascript
export {BsCart,BsHeart,SomeOther... } from "react-icons/bs";
```
Would love to have your Point of view on this though.

**Pull request merged**